app/controllers/hospitals_controller.rb

```diff
@@ -2,4 +2,30 @@ class HospitalsController < ApplicationController
   def index
     @hospitals = Hospital.all
   end
+
+  def new
+    @hospital = Hospital.new
+  end
+  
+  def create
+    unless Hospital.exists?(hospital_params)
+    
+    hospital = Hospital.create(hospital_params)
+    if hospital.persisted?
+      redirect_to hospitals_path, notice: 'Hospital added'
+    else
+      redirect_to hospitals_path, notice: 'Hospital already exists. Your hospital could not be saved'
+      end
+    end
+  end
+
+  def show
+    @hospitals = Hospital.find(params[:id])
+  end
```

Comment:
You don't have a show page. No need for this.

Reply:
I have removed  the show action